src/orient/rational-calc.orient

```diff
@@ -0,0 +1,704 @@
+      describe(fallback_ratio, "Fraction of fallback PoSts")
+      describe(fallback_period, "Time between last proof submitted and a required fallback PoSt", secs)
+      describe(fallback_period_days, "Time between last proof submitted and a required fallback PoSt", secs)
+
+      fallback_period_days = fallback_period / secs_in_day
+      posts_per_sector_per_year = eposts_per_sector_per_year + fposts_per_sector_per_year
+
+      // TODO: model single ticket vs multiple tickets
+      ElectionPoSt:
+        eposts_per_sector_per_year = blocks_in_a_year / filecoin_sectors
+
+      FallbackPoSt:
+        fposts_per_sector_per_year = (secs_in_year / fallback_period) * fallback_ratio
+
+  Mining:
+    ElectionPoSt:
+      describe(epost_challenged_sectors, "Number of challenged sectors during Election PoSt")
+      describe(epost_challenged_sectors_fraction, "Fraction of challenged sectors during Election PoSt")
+      describe(epost_data_access, "Time to access data for Election PoSt", secs)
+      describe(epost_data_access_parallel, "Time to access data for Election PoSt in parallel", secs)
+      describe(epost_time, "Time to generate an Election PoSt", secs)
+      describe(epost_time_parallel, "Time to generate an Election PoSt in parallel", secs)
+      describe(epost_inclusions_time, "Time to generate all inclusions in Election PoSt", secs)
+      describe(epost_inclusions_time_parallel, "Time to generate all the inclusions in Election PoSt in parallel", secs)
+
+      epost_challenged_sectors = rig_sectors_min * epost_challenged_sectors_fraction
+      epost_leaves_read = post_leaves_read * epost_challenged_sectors
+      epost_mtree_read = post_mtree_read
+      epost_data_access = epost_leaves_read + epost_mtree_read
+      epost_data_access_parallel = epost_data_access / rig_storage_parallelization  
+      epost_mtree_read_parallel = epost_mtree_read / rig_storage_parallelization  
+      epost_leaves_read_parallel = epost_leaves_read / rig_storage_parallelization  
+      epost_inclusions_time = post_inclusions_time * epost_challenged_sectors
+      epost_inclusions_time_parallel = epost_inclusions_time / rig_cores
+
+      epost_time = epost_data_access + post_ticket_gen + epost_inclusions_time + post_snark_time
+      epost_time_parallel = epost_data_access_parallel + post_ticket_gen + epost_inclusions_time_parallel + post_snark_time_parallel
+
+Rationality:
+  describe(breakeven_nodes, "number of nodes / sector / challenge period after which it's cheaper to store the entire sector, than re-encode those")
+  describe(polling_time, "Time between two PoSt proofs", secs)
+  describe(challenge_periods_per_month, "Number of challenge periods in a month")
+  describe(regeneration_fraction, "Fraction of nodes of a sector for which re-encoding them every `polling_time` costs as much as storing them")
+
+  breakeven_nodes = ((cost_gib_per_month *  sector_size_gib) / (cost_node_encoding * challenge_periods_per_month))
```

Comment:
The multiplication `cost_node_encoding * challenge_periods_per_month` doesn't seem to have matching units.